app/javascript/components/patient/common_exposure_cohorts/CommonExposureCohortModal.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Form, Modal } from 'react-bootstrap';
+
+const COHORT_TYPES = [
+  '',
+  'Adult Congregate Living Facility',
+  'Child Care Facility',
+  'Community Event or Mass Gathering',
+  'Correctional Facility',
+  'Group Home',
+  'Healthcare Facility',
+  'Place of Worship',
+  'School or University',
+  'Shelter',
+  'Substance Abuse Treatment Center',
+  'Workplace',
+  'Other',
+];
+
+class CommonExposureCohortModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      common_exposure_cohort: props.common_exposure_cohort || {},
+      showCohortModal: false,
+    };
+  }
+
+  handleChange = event => {
+    event.persist();
+    this.setState(state => {
+      return { common_exposure_cohort: { ...state.common_exposure_cohort, [event?.target?.id]: event?.target?.value } };
+    });
+  };
+
+  render() {
+    return (
+      <Modal size="lg" backdrop="static" show onHide={this.props.onHide} centered>
+        <Modal.Header closeButton>
+          <Modal.Title>{this.props.common_exposure_cohort ? 'Update' : 'Add'} Common Exposure Cohort</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Form.Group>
```

Comment:
add a controlId="cohort_type" here and remove the id and aria label form the control.  this makes the label associate with the control

Reply:
Wave results (should have green label icon): 
<img width="881" alt="Screen Shot 2021-11-12 at 4 06 27 PM" src="https://user-images.githubusercontent.com/35042815/141534635-39abd921-b44e-4634-8d7b-a1a8a9481ebb.png">

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1125/commits/eff7705eebef91c92d3528cf9dcba2e1dd7cd22c#diff-97fb78554e2c9392e787e0a8353b4bf0dafc28f73344f230aafc03b734078e3cR44